README.md

```diff
@@ -513,10 +514,10 @@ Requires `GITLAB_ACCESS_TOKEN` variable to be set, which is an access token with
 - Run `composer install`
 - Add your Pantheon `site-name` to the last job in the new
   workflow file at `.github/workflows/PantheonReviewApps.yml`
-- Add the following secrets to your repository:
-  - `PANTHEON_TERMINUS_TOKEN` See https://pantheon.io/docs/terminus/install#machine-token
-  - `SSH_PRIVATE_KEY` A private key of a user which can push to Pantheon
-  - `SSH_KNOWN_HOSTS` The result of running `ssh-keyscan -H codeserver.dev.$PANTHEON_SITE_ID.drush.in`
+- Add the following [variables to your GitLab repository](https://docs.gitlab.com/ee/ci/variables/#for-a-project):
+  - `PANTHEON_TERMINUS_TOKEN` See https://pantheon.io/docs/terminus/install#machine-token (enable the _Mask variable_ checkbox)
```

Comment:
Interesting. GitLab uses variables, but can mask the variable! TIL.

Reply:
I'm assuming since they were varables already for gitlab, that there isn't any change needed in the gitlab action. yeah, looks that way since https://github.com/Lullabot/drainpipe/blob/main/scaffold/gitlab/PantheonReviewApps.gitlab-ci.yml doesn't specify variable or secret.